src/routes/NotFound/NotFound.scss

```diff
@@ -1,26 +1,194 @@
-@import "constants/style.scss";
+@use "sass:color";
+@import "src/constants/style";
 
-$logo-height: 60px;
-$text-size--medium: 24px;
+.not-found__root {
+  height: 100%;
+  width: 100%;
+  padding: $padding--small 0 0 $padding--medium;
+  overflow-x: hidden;
+  background: linear-gradient(320deg, color.change($color-backlog-blue, $alpha: 0.2), color.change($color-white, $alpha: 0.3));
+}
+
+.not-found__background {
+  position: relative;
+}
+
+.not-found__background-form {
+  position: absolute;
+  top: -440px;
+  left: -960px;
+}
+
+.not-found__background-details {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+}
+
+.not-found__header {
+  position: absolute;
+}
 
-.not-found {
+.not-found__main {
   display: flex;
-  flex-direction: column;
+  flex-direction: column-reverse; // in mobile, stan is above the content
+  gap: 0;
+
+  height: 100%;
+
   justify-content: center;
   align-items: center;
-  height: 100%;
-  width: 100%;
 }
 
-.not-found__logo {
-  height: $logo-height;
+// content (left)
+.not-found__content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  padding: 0 $padding--default;
+
+  z-index: $base-z-index; // layer content over background, using any z-index works
 }
 
-.not-found__text {
-  color: $color-black;
-  font-size: $text-size--medium;
-  letter-spacing: $letter-spacing--medium;
+.not-found__title {
+  font-weight: 700;
+  font-size: 45px;
+}
+
+.not-found__description {
+  margin-top: $margin--small;
+
+  font-weight: 400;
+  font-size: 24px;
+  line-height: 30px;
 }
-[theme="dark"] .not-found__text {
+
+.not-found__return-button {
+  margin-top: 20px;
+
+  padding: 15px 30px;
   color: $color-white;
+  background: #ff0069;
+
+  font-weight: 700;
+  font-size: $text-size--large;
```

Comment:
In my opinion either the font size or the button in general looks too big
<img width="434" alt="image" src="https://user-images.githubusercontent.com/36969812/236422505-c8c8c1cb-bad1-4c2d-9258-f11aecd8bcb2.png">

Reply:
I think it looks better with `font-size: 16px`